verdigris/rules.mk

```diff
@@ -0,0 +1,25 @@
+# Phaethon - A FLOSS resource explorer for BioWare's Aurora engine games
+#
+# Phaethon is the legal property of its developers, whose names
+# can be found in the AUTHORS file distributed with this source
+# distribution.
+#
+# Phaethon is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# Phaethon is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Phaethon. If not, see <http://www.gnu.org/licenses/>.
+
+# Verdigris so we don't need moc (<https://github.com/woboq/verdigris>).
+
+noinst_HEADERS += \
+    verdigris/wobjectdefs.h \
+    verdigris/wobjectimpl.h \
+    $(EMPTY)
```

Comment:
Can you also add a README.Phaethon file into the verdigris directory, with the contents
```
This is the source of Verdigris (<https://github.com/woboq/verdigris>),
based on the git revision .
```
(and the revision you based it on filled in there).

Also, add a section for it in the AUTHORS file (probably right after UTF-8 CPP):
```
Verdigris
*********
   Phaethon uses and includes the header-only library Verdigris
   (<https://github.com/woboq/verdigris>) to use Qt5 without moc. Verdigris
   is licensed under version 3 or later of the Lesser GNU General Public
   License. For the full license texts, see LICENSE.LGPLv3 or
   <https://gnu.org/licenses/lgpl-3.0.html>.
```

And drop in that license file in plaintext.

Likewise, add the two Verdigris files to dists/debian/copyright:
```
Files: verdigris/wobjectdefs.h
       verdigris/wobjectimpl.h
Copyright: 2016-2018 Woboq GmbH 
License: LGPL-3.0+
```
And you then need a section
```
License: LGPL-3.0+
 This package is free software; you can redistribute it and/or
 modify it under the terms of the GNU Lesser General Public
 License as published by the Free Software Foundation; either
 version 3 of the License, or (at your option) any later version.
 .
 This package is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 Lesser General Public License for more details.
 .
 On Debian systems, the complete text of the GNU Library General Public License
 can be found in the file "/usr/share/common-licenses/LGPL-3".

```
at the bottom of the copyright file.

This should all go within this commit, please.